### PR TITLE
Pass conf to export function

### PR DESCRIPTION
**Pull request by craigmulligan**
Connects to #2954

### CODE REVIEW

File: server/export.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ import { renderToHTML } from './render'
 import { getAvailableChunks } from './utils'
 import { printAndExit } from '../lib/utils'
 
-export default async function (dir, options) {
+export default async function (dir, options, conf) {
```
Comment (timneutkens) [inline server/export.js]: Lets call this `configuration`

Comment (craig-mulligan) [reply server/export.js]: Sure 👍

**Review by timneutkens — APPROVED** [review]
Seems alright 👍 

cc @arunoda

**Comment by timneutkens** [comment]
Thanks!

**Pull request merged**